__init__.py

```diff
@@ -159,21 +103,128 @@ def handleQuery(query):
             binary = binaries[app_name]
             if not binary:
                 continue
-            
+
             executable = binary[0]
             icon = binary[1]
-            
-            output_entry = Item(
-                id="%015d-%s-%s" % (now - last_update, project_path, app_name),
-                icon=icon,
-                text=project_dir,
-                subtext=project_path,
-                completion=__triggers__ + project_dir,
-                actions=[
-                    ProcAction(text="Open in %s" % app_name, commandline=[executable, project_path])
-                ]
+
+            items.append(
+                Item(
+                    id="%015d-%s-%s" % (now - last_update, project_path, app_name),
+                    text=project_dir,
+                    subtext=project_path,
```

Comment:
Maybe change the subtext to also include the app (since it is no longer obvious from the logo? hmmm. What do you think?

Reply:
I changed the subtext to "Open $project_path with $app_name".